pytket/binders/transform.cpp

```diff
@@ -48,7 +48,7 @@ Transforms::TwoQbFidelities get_fidelities(const py::kwargs &kwargs) {
     } else if (kwargstr == "ZZMax_fidelity") {
       fid.ZZMax_fidelity = py::cast<double>(kwarg.second);
     } else if (kwargstr == "ZZPhase_fidelity") {
-      fid.ZZPhase_fidelity = py::cast<Func>(kwarg.second);
+      fid.ZZPhase_fidelity = py::cast<std::variant<double, Func>>(kwarg.second);
```

Comment:
Please update the docstring for `Transform.DecomposeTK2` below to reflect this new possible usage.